core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java

```diff
@@ -41,74 +44,18 @@
 
-	private static final String FUSE_621_R0_CAMEL_VERSION = "2.15.1.redhat-621084";
-	private static final String FUSE_621_R1_CAMEL_VERSION = "2.15.1.redhat-621090";
-	private static final String FUSE_621_R2_CAMEL_VERSION = "2.15.1.redhat-621107";
-	private static final String FUSE_621_R3_CAMEL_VERSION = "2.15.1.redhat-621117";
-	private static final String FUSE_621_R4_CAMEL_VERSION = "2.15.1.redhat-621159";
-	private static final String FUSE_621_R5_CAMEL_VERSION = "2.15.1.redhat-621169";
-	private static final String FUSE_621_R6_CAMEL_VERSION = "2.15.1.redhat-621177";
-	private static final String FUSE_621_R7_CAMEL_VERSION = "2.15.1.redhat-621186";
-	private static final String FUSE_621_R8_CAMEL_VERSION = "2.15.1.redhat-621195";
-	private static final String FUSE_621_R9_CAMEL_VERSION = "2.15.1.redhat-621216";
-	
-	private static final String FUSE_621_R0_BOM_VERSION = "6.2.1.redhat-084";
-	private static final String FUSE_621_R1_BOM_VERSION = "6.2.1.redhat-090";
-	private static final String FUSE_621_R2_BOM_VERSION = "6.2.1.redhat-107";
-	private static final String FUSE_621_R3_BOM_VERSION = "6.2.1.redhat-117";
-	private static final String FUSE_621_R4_BOM_VERSION = "6.2.1.redhat-159";
-	private static final String FUSE_621_R5_BOM_VERSION = "6.2.1.redhat-169";
-	private static final String FUSE_621_R6_BOM_VERSION = "6.2.1.redhat-177";
-	private static final String FUSE_621_R7_BOM_VERSION = "6.2.1.redhat-186";
-	private static final String FUSE_621_R8_BOM_VERSION = "6.2.1.redhat-195";
-	private static final String FUSE_621_R9_BOM_VERSION = "6.2.1.redhat-216";
-	
-	private static final String FUSE_63_R0_CAMEL_VERSION = "2.17.0.redhat-630187";
-	private static final String FUSE_63_R1_CAMEL_VERSION = "2.17.0.redhat-630224";
-	private static final String FUSE_63_R2_CAMEL_VERSION = "2.17.0.redhat-630254";
-	private static final String FUSE_63_R3_CAMEL_VERSION = "2.17.0.redhat-630262";
-	public static final String FUSE_63_R4_CAMEL_VERSION = "2.17.0.redhat-630283";
-	public static final String FUSE_63_R5_CAMEL_VERSION = "2.17.0.redhat-630310";
-
-	private static final String FUSE_63_R0_BOM_VERSION = "6.3.0.redhat-187";
-	private static final String FUSE_63_R1_BOM_VERSION = "6.3.0.redhat-224";
-	private static final String FUSE_63_R2_BOM_VERSION = "6.3.0.redhat-254";
-	private static final String FUSE_63_R3_BOM_VERSION = "6.3.0.redhat-262";
-	static final String FUSE_63_R4_BOM_VERSION = "6.3.0.redhat-283";
-	static final String FUSE_63_R5_BOM_VERSION = "6.3.0.redhat-310";
-	
-	public static final String FIS_20_R1_CAMEL_VERSION = "2.18.1.redhat-000012";
-	public static final String FIS_20_R2_CAMEL_VERSION = "2.18.1.redhat-000015";
-	public static final String FIS_20_R3_CAMEL_VERSION = "2.18.1.redhat-000021";
-	
-	public static final String DEFAULT_CAMEL_VERSION = FIS_20_R3_CAMEL_VERSION;
+	public static final String DEFAULT_CAMEL_VERSION = CamelForFIS6ToBomMapper.FIS_20_R3_CAMEL_VERSION;
```

Comment:
that assignment looks a bit weird at first sight. but I guess thats fine for now.

Reply:
What do you find weird in it? That it is taking a value from CamelForFIS6ToBomMapper ? or that it is the FIS 2.0 R3?

Reply:
actually both :D

Reply:
wondering if we should define that value directly in that class

Reply:
I prefer to let it here for now. The default Camel version is "cross-Mappers".